### PR TITLE
homebrew: cask aliases update

### PR DESCRIPTION
**Pull request by indrajitr**
- cask aliases update
- Remove wrapper function for deprecation warning
- Add alias `casku`
- Documentation update

Supersedes #1879

/cc @Rawmanski

### CODE REVIEW

**Review by belak — APPROVED** [review]
LGTM!

File: modules/homebrew/README.md [hunk 1]
```diff
@@ -23,15 +23,16 @@ Aliases
   - `brewl` lists installed formulae.
   - `brewo` lists brews which have an update available.
   - `brews` searches for a formula.
-  - `brewu` updates and upgrades Homebrew packages and formulae.
```
Comment (belak) [inline modules/homebrew/README.md:26]: I'm not 100% sure this is a readability improvement but I get what it's trying to do so it's probably fine

File: modules/homebrew/README.md [hunk 2]
```diff
@@ -23,15 +23,16 @@ Aliases
...
+  - `brewu` updates Homebrew and upgrades formulae.
```
Comment (indrajitr) [inline modules/homebrew/README.md:26]:
```suggestion
  - `brewu` upgrades outdated formulae.
```

File: modules/homebrew/README.md (hunk 1 above)
Comment (indrajitr) [reply modules/homebrew/README.md:26]: Agreed! All `brew` operations update Homebrew implicitly anyway. How does the suggested change sound?

File: modules/homebrew/README.md [hunk 3]
```diff
@@ -23,15 +23,16 @@ Aliases
...
   - `brewx` uninstalls a formula.
 
 ### Homebrew Cask
 
-  - `cask` is aliased to `brew cask`.
   - `caski` installs a cask.
   - `caskl` lists installed casks.
   - `casko` lists casks which have an update available.
+  - `casks` searches for a cask.
+  - `casku` updates Homebrew and upgrades casks.
```
Comment (indrajitr) [inline modules/homebrew/README.md:35]:
```suggestion
  - `casku` upgrades outdated casks.
```

**Review by indrajitr — COMMENTED** [review]
@belak PTAL

**Comment by indrajitr** [comment]
Also closes #1866

**Pull request merged**